src/components/form/Lookup/User/UserField.tsx

```diff
@@ -1,13 +1,27 @@
+import { CreatePersonInput } from '../../../../api';
+import { CreateUser } from '../../../../scenes/Users/Create';
 import { LookupField } from '../../index';
 import {
   UserLookupItemFragment,
   useUserLookupLazyQuery,
 } from './UserLookup.generated';
 
-export const UserField = LookupField.createFor<UserLookupItemFragment>({
+export const UserField = LookupField.createFor<
+  UserLookupItemFragment,
+  CreatePersonInput
+>({
   resource: 'User',
   useLookup: useUserLookupLazyQuery,
   getOptionLabel: (option) => option.fullName,
   label: 'Person',
   placeholder: 'Search for a person by name',
+  CreateDialogForm: CreateUser,
+  getInitialValues: (val): Partial<CreatePersonInput> => ({
+    // @ts-expect-error the partial type doesn't match and the generic is not
+    // being passed around everywhere yet.
+    person: {
+      realFirstName: val,
```

Comment:
I was thinking that val should be split by whitespace and populate first and last names. If nothing after whitespace, then just do first name.

Reply:
```typescript
getInitialValues: (val): Partial<CreatePersonInput> => {
  const [first, last] = val.split(' ');
  return { 
    person: {
      realFirstName: first,
      realLastName: last,
      displayFirstName: first,
      displayLastName: last,
    }
  }
}



```